src/bls_ops.rs

```diff
@@ -240,10 +243,13 @@ pub fn op_bls_map_to_g1(a: &mut Allocator, input: NodePtr, max_cost: Cost) -> Re
 pub fn op_bls_map_to_g2(a: &mut Allocator, input: NodePtr, max_cost: Cost) -> Response {
     let ([msg, dst], argc) = get_varargs::<2>(a, input, "g2_map")?;
     if !(1..=2).contains(&argc) {
-        return err(input, "g2_map takes exactly 1 or 2 arguments");
+        Err(EvalErr::InvalidArg(
+            input,
+            format!("G2_map takes exactly 1 or 2 arguments, got {argc}"),
```

Comment:
```suggestion
            format!("g2_map takes exactly 1 or 2 arguments, got {argc}"),
```
This operator is called `g2_map`, with lower case "g".